front/app/containers/Admin/messaging/texting/NewCampaign/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from 'react';
+
+// components
+import TextArea from 'components/UI/TextArea';
+import Button from 'components/UI/Button';
+import { Label, Box, IconTooltip } from '@citizenlab/cl2-component-library';
+import { Section, SectionField } from 'components/admin/Section';
+import HelmetIntl from 'components/HelmetIntl';
+import TextingHeader from '../TextingHeader';
+
+// services
+import { addTextingCampaign } from 'services/textingCampaigns';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+
+// styling
+import styled from 'styled-components';
+
+const StyledForm = styled.form`
+  width: 500px;
+`;
+
+// enough to fit 3 messages, actual functionality TBD in subsequent ticket
+const MAX_CHAR_COUNT = 480;
+
+const TextCreation = () => {
+  const [inputPhoneNumbers, setInputPhoneNumbers] = useState('');
+  const [inputMessage, setInputMessage] = useState('');
```

Comment:
I'd rather have null as a begin value to be consistent with the rest of the codebase. If that's no issue with the prop types.